entries/d/datev.de.json

```
@@ -0,0 +1,15 @@
+{
+  "DATEV": {
+    "domain": "datev.de",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://apps.datev.de/help-center/documents/1022936",
```

Would this be a correct description?
```suggestion
    "documentation": "https://apps.datev.de/help-center/documents/1022936",
    "notes": "2FA only available for DATEV Arbeitnehmer online",
```